aip/general/0121.md

```diff
@@ -61,6 +61,10 @@ exposes a large number of resources with a small number of methods on each
 resource. The methods can be either the standard methods ([Get][], [List][],
 [Create][], [Update][], [Delete][]), or [custom methods][].
 
+A resource that supports a [Create][], [Update][], or [Delete][] must also
+support [Get][] to enable the user to validate the success of their operation
```

Comment:
"user" --> "client"?

Reply:
a user may also want to verify? but I'm not strongly opinionated. 

do we want to normalize such nomenclature? I imagine "consumer" is another one.